### ISSUE TITLE
[DataGrid] Group By Sorting Event

### ISSUE THREAD

**Issue by anserban** [IgniteUI/igniteui-angular#12447]
I'm using a noop sorting strategy for data grid. Once the user performs group by, I'd be interested in listening for when he tries to change the sorting from the grouping panel above in order to do this action on the server side, but I couldn't find any event method for this particular case.

I've tried to subscribe to `groupingExpressionsChange` event and expected to catch it there, but it does not.
Is there something available for this in the latest version or could it be added in a future one?

igniteui-angular version: 14.2.6
browser: Chrome

**Comment by dkamburov** [comment]
@anserban You're right clicking on a grouping chip doesn't trigger `sortingExpressionsChange` or `groupingExpressionsChange`.  We'll investigate this further and reach back to you.

This is the samples I've used for testing it - https://stackblitz.com/edit/angular-gx8scb-ojfqmy?file=src/app/grid/grid-remote-filtering-sample/remote-filtering-sample.component.html
